app/lib/common/widget/CrossSigning.dart

```diff
@@ -5,7 +5,7 @@ import 'dart:async';
 import 'package:effektio/common/store/separatedThemes.dart';
 import 'package:effektio/common/widget/AppCommon.dart';
 import 'package:effektio_flutter_sdk/effektio_flutter_sdk_ffi.dart'
-    show SyncState, CrossSigningEvent, Client, FfiListEmojiUnit;
+    show Client, FfiListEmojiUnit, GenericEvent, SyncState;
```

Comment:
Using the different class here. Does this handle both emoji and other events? 
Note: Just for my reminder, the PR push can break the implemented cross-signing UI and I need to fix it.